x-pack/plugins/observability_solution/observability_onboarding/public/application/experimental_onboarding_flow.tsx

```diff
@@ -22,6 +22,13 @@ import { CustomLogsPanel } from './quickstart_flows/custom_logs';
 const queryClient = new QueryClient();
 
 export function ExperimentalOnboardingFlow() {
+  const mainContentRef = useRef<HTMLDivElement | null>(null);
+  const location = useLocation();
+
+  useEffect(() => {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
**nit**: the smooth behaviour feels a bit weird when switching to a completely new page, I would recommend using the suggested `window.scrollTo(0, 0);` from the [docs](https://v5.reactrouter.com/web/guides/scroll-restoration/scroll-to-top).

Reply:
True, this will look better 👍